http/src/main/resources/reference.conf

```diff
@@ -153,6 +153,9 @@ proxy {
   networkTag = "leonardo"
   projectVPCNetworkLabel = "vpc-network-name"
   projectVPCSubnetLabel = "vpc-subnetwork-name"
+  # Subnets are regional and independent of the runtime region. Networks on the other hand are global.
```

Comment:
Does ^^ contradict ```Because subnets are regional objects, the region that you select for a resource determines the subnets that it can use``` from the link below?

Reply:
Ugh yeah good catch. Maybe instead of this config it should just use the region of the cluster/instance.

Reply:
That sounds better. I'd think this is fine for now since we're creating resources only in `us-central1` but may break if we allow other regions.

Reply:
I think I will update the PR to do that